doc/whats_new.rst

```diff
@@ -37,6 +37,8 @@ BUG
 
     - Time cropping functions no longer use relative tolerances when determining the boundaries by `Jaakko Leppakangas`_
 
+    - Fix :func:`mne.decoding.TimeDecoding` to allow specifying `clf` by `Jean-Remi King`_
```

Comment:
the single-backticks on `clf` here won't be made into a proper reference, so better to use double-backticks so it gets rendered as code